src/gnome-utils/ui/gnucash-fallback.css

```diff
@@ -2,3 +2,16 @@
    directly by name as these are not brought in when loaded, only
    the widget type can be configured unless they are named in code */
 
+
+/* These are the defaults used in the dense-cal */
+*.primary {
+  background-color: lavender;
+}
+
+*.secondary {
+  background-color: SlateGray1;
+}
+
+*.markers {
+  background-color: yellow;
+}
```

Comment:
If we want to keep these as default colors, we'll have to set text colors as well to match them. In the dark theme the text color is white, which will be pretty hard to read.

I'd propose however  let go of these colors and instead expermiment with colors that blend in with the theme like background color +/-10%. Or if we must have a gnucash specific style, we can try to reuse the yellow/green theme also used in the register.
These gnucash specific colors can be activated/deactivated via the same single preference we already have for the register. In that case the preference should become more generic of course and probably be presented in a different preference tab.